Food_Tracker_App/app/src/main/java/com/example/foodtracker/Category.java

```diff
@@ -0,0 +1,88 @@
+package com.example.foodtracker;
+
+import android.os.Build;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.List;
+
+/**
+ * category for ingredients
+ * returns the categories by getCategories
+ */
+public enum Category {
```

Comment:
For both Location and Category we will need some capacity to add constants

Reply:
I don't know if an enum is the best way to do this since the enum is static at compile time...